erp-playbook/requirements.txt

```diff
@@ -1,3 +1,3 @@
-ansible==2.5.4
+ansible>=2.5.5
 future==0.16.0
-requests==2.18.4
+requests>=2.20
```

Comment:
```suggestion
requests==2.20.0
```

Reply:
The file is mainly used to make sure newer version pkgs don't break CI setup, so we need fixed versions here.